lib/commands/std-config-options-read.ts

```diff
@@ -0,0 +1,17 @@
+/* Copyright (c) 2024. SailPoint Technologies, Inc. All rights reserved. */
+
+import {ConfigOptions, ObjectInput, ObjectOutput} from './command'
+
+/**
+ * Input object of `std:config-options:read` command
+ */
+export type StdConfigOptionsInput = ObjectInput & {
+    key: string
+}
+
+/**
+ * Output object of `std:config-options:read` command
+ */
+export type StdConfigOptionsOutput = ObjectOutput & {
+	output: ConfigOptions
+}
```

Comment:
I don't think we need the intersections with `ObjectInput` and `ObjectOutput` here.

We might also want to consider renaming `output` to something like `options` to make it more descriptive.

Reply:
Can you please elaborate this 
> I don't think we need the intersections with ObjectInput and ObjectOutput here.

Reply:
I think it could just be:

```typescript
export type StdConfigOptionsInput = {
    key: string
}

export type StdConfigOptionsOutput = {
	output: ConfigOptions
}
```

Reply:
done